onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -193,240 +192,286 @@ static bool ParseDimensionOverride(std::basic_string<ORTCHAR_T>& dim_identifier,
+    options.add_options()("t", "", cxxopts::value<size_t>());
+    options.add_options()("p", "", cxxopts::value<std::string>());
+    options.add_options()("x", "", cxxopts::value<int>());
+    options.add_options()("y", "", cxxopts::value<int>());
+    options.add_options()("c", "", cxxopts::value<size_t>());
+    options.add_options()("d", "", cxxopts::value<int>());
+    options.add_options()("o", "", cxxopts::value<int>());
+    options.add_options()("u", "", cxxopts::value<std::string>());
+    options.add_options()("i", "", cxxopts::value<std::string>());
+    options.add_options()("S", "", cxxopts::value<int>());
+    options.add_options()("T", "", cxxopts::value<std::string>());
+    options.add_options()("C", "", cxxopts::value<std::string>());
+    options.add_options()("R", "", cxxopts::value<std::string>());
+    options.add_options()("A", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("M", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("s", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("v", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("I", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("P", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("q", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("z", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("D", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("Z", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("n", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("l", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("g", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("X", "", cxxopts::value<bool>()->default_value("false")->implicit_value("true"));
+    options.add_options()("plugin_ep_libs", "", cxxopts::value<std::string>());
+    options.add_options()("list_ep_devices", "");
+    options.add_options()("select_ep_devices", "", cxxopts::value<std::string>());
+    options.add_options()("h,help", "");
+
+#ifdef _WIN32
+    auto utf8_strings = utils::ConvertArgvToUtf8Strings(argc, argv);
+    auto utf8_argv = utils::CStringsFromStrings(utf8_strings);
+    auto result = options.parse(static_cast<int>(utf8_argv.size()), utf8_argv.data());
+#else
+    auto result = options.parse(argc, argv);
+#endif
+
+    if (result.count("f")) {
+      std::basic_string<ORTCHAR_T> dim_name;
```

Comment:
since we are converting to utf8 strings first on Windows, maybe we can simplify the string handling in general and just use utf8 std::strings. the exception to this would be for actual file paths, like the model path.

Reply:
That's a good point.
Looking at test_configuration.h, `free_dim_name_overrides `and `free_dim_denotation_overrides` can be changed to use utf8 std::string, and i just updated them.  Other string-based options all have to deal with file paths, so i will leave them as `std::basic_string<ORTCHAR_T>`